plugins/code-coverage/src/components/FileExplorer/FileExplorer.tsx

```diff
@@ -232,7 +232,7 @@ export const FileExplorer = () => {
       title: 'Coverage',
       type: 'numeric',
       field: 'coverage',
-      render: (row: CoverageTableRow) => `${row.coverage}%`,
+      render: (row: CoverageTableRow) => `${row.coverage.toFixed(2)}%`,
```

Comment:
random query: this obviously fixes it when rendering, but the value in the object itself is still at a higher precision. for the purposes of the plugin, should the precision be limited when adding to the object initially? that way, if this plugin is later modified to add more panels or displays of the data, the data itself is just always simply at a lower precision? that would avoid having to add `.toFixed(2)` elsewhere throughout the entire UI, and to be honest within Backstage I'd guess having further precision would never be necessary. (even if the code coverage app ITSELF might allow that higher precision)

not sure if that makes sense or not!

Reply:
Hmm, I understand what you're saying and it does make sense. Looking through the code it appears the `calculatePercentage` func already does the `.toFixed(2)` and I'm not finding where else the coverage percentage is calculated at and why I needed to add it when rendering the table. I'm still  looking for a better location to round the data.

Reply:
I see the percentage is calculated on-the-fly in buildFileStructure(), so I'm not sure there is a better place to make this change besides the one I used. Though I'm totally open to suggestions.

Reply:
Updated PR.

UI dates now look like
<img width="1477" alt="Screen Shot 2021-11-08 at 9 20 36 AM" src="https://user-images.githubusercontent.com/23618053/140789835-755ec39a-3fbe-41eb-8db1-33c347b83939.png">